[3.0.14] bbb-web hits `nofile` limit on machines with very high core count

<!--PLEASE DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
This issue tracker is only for bbb development related issues.-->

**Describe the bug**
With 3.0.14, bbb-web doesn't start anymore on a machine with 96 cores, due to hitting open file limits.

**To Reproduce**
Steps to reproduce the behavior:
1. Use a system with very high CPU count (observed on system with 96 hardware threads)
2. Start BigBluebutton
3. BigBlueButton doesn't start
4. See logs below

**Expected behavior**
bbb-web starts up fine

**Actual behavior**
bbb-web doesn't start, because netty causes it to hit open file limit. `/var/log/bigbluebutton/bbb-web.log`:

```
2025-09-14T09:33:54.163+02:00 INFO  org.bigbluebutton.web.Application - The following 1 profile is active: "production"
2025-09-14T09:33:55.942+02:00 INFO  o.b.w.services.turn.StunTurnService - Reading stun/turn server config from overlay config file /etc/bigbluebutton/turn-stun-servers.xml
2025-09-14T09:33:58.622+02:00 INFO  o.b.a.m.ReceivedMessageHandler - Ready to handle messages from Redis pubsub!
2025-09-14T09:33:59.243+02:00 INFO  o.b.c.redis.RedisStorageService - Starting RedisStorageService with client name: clientName=BbbWebRedisStore
2025-09-14T09:33:59.680+02:00 INFO  o.b.c.redis.RedisStorageService - Connected to redis. clientName=BbbWebRedisStore
2025-09-14T09:33:59.704+02:00 INFO  o.b.c.redis.RedisStorageService - Connected to redis activated. clientName=BbbWebRedisStore
2025-09-14T09:33:59.765+02:00 INFO  o.b.common2.redis.RedisPublisher - Connected to redis. clientName=BbbWebPub
2025-09-14T09:33:59.765+02:00 INFO  o.b.common2.redis.RedisPublisher - Connection to redis activated. clientName=BbbWebPub
2025-09-14T09:33:59.772+02:00 INFO  o.b.common2.redis.RedisPublisher - Connected to redis. clientName=BbbWebPub
2025-09-14T09:33:59.774+02:00 INFO  o.b.common2.redis.RedisPublisher - Connection to redis activated. clientName=BbbWebPub
2025-09-14T09:33:59.777+02:00 DEBUG o.b.api2.bus.MessageSenderActor - started (org.bigbluebutton.api2.bus.MessageSenderActor@b722c56)
2025-09-14T09:33:59.779+02:00 DEBUG o.b.a.m.OldMeetingMsgHdlrActor - started (org.bigbluebutton.api2.meeting.OldMeetingMsgHdlrActor@45ee7861)
2025-09-14T09:33:59.784+02:00 DEBUG o.b.a.bus.MsgToAkkaAppsToJsonActor - started (org.bigbluebutton.api2.bus.MsgToAkkaAppsToJsonActor@607c2cbf)
2025-09-14T09:33:59.791+02:00 DEBUG o.b.a.b.OldMessageJsonReceiverActor - started (org.bigbluebutton.api2.bus.OldMessageJsonReceiverActor@2f6864b0)
2025-09-14T09:33:59.791+02:00 DEBUG o.b.a.bus.ReceivedJsonMsgHdlrActor - started (org.bigbluebutton.api2.bus.ReceivedJsonMsgHdlrActor@6cf9b76)
2025-09-14T09:33:59.800+02:00 INFO  o.b.a.e.r.WebRedisSubscriberActor - Connected to redis. clientName=BbbWebSub
2025-09-14T09:33:59.803+02:00 INFO  o.b.a.e.r.WebRedisSubscriberActor - Connection to redis activated. clientName=BbbWebSub
2025-09-14T09:33:59.812+02:00 DEBUG o.b.a.e.r.WebRedisSubscriberActor - started (org.bigbluebutton.api2.endpoint.redis.WebRedisSubscriberActor@126d4f62)
2025-09-14T09:33:59.815+02:00 INFO  o.b.a.e.r.WebRedisSubscriberActor - Subscribed to pattern from-akka-apps-redis-channel
2025-09-14T09:33:59.818+02:00 INFO  o.b.c.redis.RedisStorageService - Starting RedisStorageService with client name: clientName=BbbWeb
2025-09-14T09:34:01.884+02:00 ERROR o.s.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'meetingService' defined in class path resource [spring/resources.xml]: Cannot resolve reference to bean 'redisStorageService' while setting bean property 'redisStorageService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'redisStorageService' defined in class path resource [spring/bbb-redis-messaging.xml]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: failed to create a child event loop
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:342)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:113)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1707)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1452)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:929)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:591)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:147)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:409)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:99)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:485)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:472)
	at grails.boot.GrailsApp$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)
	at org.bigbluebutton.web.Application.main(Application.groovy:8)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'redisStorageService' defined in class path resource [spring/bbb-redis-messaging.xml]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: failed to create a child event loop
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1804)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:330)
	... 32 common frames omitted
Caused by: java.lang.IllegalStateException: failed to create a child event loop
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:88)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:60)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:49)
	at io.netty.channel.MultithreadEventLoopGroup.<init>(MultithreadEventLoopGroup.java:59)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:87)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:82)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:69)
	at io.lettuce.core.resource.DefaultEventLoopGroupProvider.createEventLoopGroup(DefaultEventLoopGroupProvider.java:239)
	at io.lettuce.core.resource.DefaultEventLoopGroupProvider.doCreateEventLoopGroup(DefaultEventLoopGroupProvider.java:190)
	at io.lettuce.core.resource.DefaultEventLoopGroupProvider.getOrCreate(DefaultEventLoopGroupProvider.java:172)
	at io.lettuce.core.resource.DefaultEventLoopGroupProvider.allocate(DefaultEventLoopGroupProvider.java:113)
	at io.lettuce.core.AbstractRedisClient.lambda$getEventLoopGroup$0(AbstractRedisClient.java:305)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at io.lettuce.core.AbstractRedisClient.getEventLoopGroup(AbstractRedisClient.java:304)
	at io.lettuce.core.ConnectionBuilder.configureBootstrap(ConnectionBuilder.java:258)
	at io.lettuce.core.AbstractRedisClient.connectionBuilder(AbstractRedisClient.java:274)
	at io.lettuce.core.RedisClient.connectStatefulAsync(RedisClient.java:322)
	at io.lettuce.core.RedisClient.connectStandaloneAsync(RedisClient.java:287)
	at io.lettuce.core.RedisClient.connect(RedisClient.java:216)
	at io.lettuce.core.RedisClient.connect(RedisClient.java:201)
	at org.bigbluebutton.common2.redis.RedisStorageService.start(RedisStorageService.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1930)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1872)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800)
	... 39 common frames omitted
Caused by: io.netty.channel.ChannelException: failed to open a new selector
	at io.netty.channel.nio.NioEventLoop.openSelector(NioEventLoop.java:179)
	at io.netty.channel.nio.NioEventLoop.<init>(NioEventLoop.java:146)
	at io.netty.channel.nio.NioEventLoopGroup.newChild(NioEventLoopGroup.java:183)
	at io.netty.channel.nio.NioEventLoopGroup.newChild(NioEventLoopGroup.java:38)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:84)
	... 66 common frames omitted
Caused by: java.io.IOException: Too many open files
	at java.base/sun.nio.ch.EventFD.eventfd0(Native Method)
	at java.base/sun.nio.ch.EventFD.<init>(EventFD.java:40)
	at java.base/sun.nio.ch.EPollSelectorImpl.<init>(EPollSelectorImpl.java:82)
	at java.base/sun.nio.ch.EPollSelectorProvider.openSelector(EPollSelectorProvider.java:36)
	at io.netty.channel.nio.NioEventLoop.openSelector(NioEventLoop.java:177)
	... 70 common frames omitted
```

**Additional context**
Running on 2x AMD EPYC 7413 24-Core Processor . Many settings are customized, as this is a special "bigconf" system. Nevertheless, this issue is definitely new in 3.0.14 vs 3.0.13 without relevant configuration changes in between.

**ONE WAY TO FIX IT:**
Create/edit `/etc/systemd/system/bbb-web.service.d/override.conf ` and set

```
[Service]
LimitNOFILE=4096
```

--- a/build/packages-template/bbb-web/bbb-web.service
+++ b/build/packages-template/bbb-web/bbb-web.service
@@ -19,8 +19,7 @@ RestartSec=60
 SuccessExitStatus=143
 TimeoutStopSec=5
 PermissionsStartOnly=true
-LimitNOFILE=1024
+LimitNOFILE=4096
 
 [Install]
 WantedBy=multi-user.target bigbluebutton.target
-

